src/unix/linux_like/mod.rs

```diff
@@ -775,6 +775,11 @@ pub const MSG_ERRQUEUE: ::c_int = 0x2000;
 pub const MSG_NOSIGNAL: ::c_int = 0x4000;
 pub const MSG_MORE: ::c_int = 0x8000;
 pub const MSG_WAITFORONE: ::c_int = 0x10000;
+cfg_if! {
+    if #[cfg(not(all(target_os = "emscripten", target_os = "android")))] {
```

Comment:
Could you just move them to src/unix/linux_like/linux/mod.rs to avoid cfg complexity?

Reply:
ah you re right